tools/run_tests/sanity/check_submodules.sh

```diff
@@ -38,7 +38,7 @@ cat << EOF | awk '{ print $1 }' | sort > "$want_submodules"
  ec44c6c1675c25b9827aacd08c02433cccde7780 third_party/googletest (release-1.8.0)
  6599cac0965be8e5a835ab7a5684bbef033d5ad0 third_party/libcxx (heads/release_60)
  9245d481eb3e890f708ff2d7dadf2a10c04748ba third_party/libcxxabi (heads/release_60)
- 48cb18e5c419ddd23d9badcfe4e9df7bde1979b2 third_party/protobuf (v3.6.0.1-37-g48cb18e5)
+ 582743bf40c5d3639a70f98f183914a2c0cd0680 third_party/protobuf (v3.6.0.1-37-g48cb18e5)
```

Comment:
nit: probably need to update `(v3.6.0.1-37-g48cb18e5)` to the new tag... Also, since it looks like these git ref's in parentheses are unused, maybe they can be removed, or placed in comments.

Reply:
Done.